src/DripsHub.sol

```diff
@@ -103,14 +103,14 @@ abstract contract DripsHub {
     /// @param receivers The new list of the sender's receivers.
     event SenderUpdated(address indexed sender, uint128 balance, Receiver[] receivers);
 
-    /// @notice Emitted when a sub-sender is updated
+    /// @notice Emitted when a sender account is updated
     /// @param senderAddr The address of the sender
-    /// @param subSenderId The id of the sender's updated sub-sender
-    /// @param balance The sub-sender's balance
-    /// @param receivers The new list of the sub-sender's receivers.
+    /// @param account The sender's account
+    /// @param balance The account's balance
+    /// @param receivers The new list of the account's receivers.
     event SenderUpdated(
```

Comment:
We still call the event Sender here so i guess it is fine?

Reply:
We'll be dropping the notion of a `sending` in favor of dripping, so `sender` will go too. E.g. `event SenderUpdated` will be `event DripsUpdated`.

Reply:
okay